ngraph_bridge/ngraph_builder.cc

```diff
@@ -3186,6 +3186,36 @@ static Status TranslateRankOp(const Node* op, const std::vector<const Tensor*>&,
   return Status::OK();
 }
 
+static Status TranslateRandomUniformOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  // std::cout<<"--------------- begin\n";
+  shared_ptr<ng::Node> ng_input;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_input));
+
+  auto const_min = ConstructNgNode<ng::op::Constant>(
+      op->name(), ng::element::f32, ng::Shape{}, std::vector<float>{0});
+
+  auto const_max = ConstructNgNode<ng::op::Constant>(
+      op->name(), ng::element::f32, ng::Shape{}, std::vector<float>{1});
+
+  std::vector<int64> shape;
+  TF_RETURN_IF_ERROR(GetStaticInputVector(op, 0, static_input_map, &shape));
+
+  auto const_shape = ConstructNgNode<ng::op::Constant>(
+      op->name(), ng::element::i64, ng::Shape{shape.size()}, shape);
+
+  auto const_use_fixed_seed = ConstructNgNode<ng::op::Constant>(
+      op->name(), ng::element::boolean, ng::Shape{}, std::vector<bool>{false});
+
+  auto random_uniform = ConstructNgNode<ng::op::RandomUniform>(
+      op->name(), const_min, const_max, const_shape, const_use_fixed_seed, 0);
```

Comment:
Maybe we need to read the Seed from the TF op and pass it to ngraph's `RandomUniform` instead of `0`.

Even then do we expect TF's random number and ngraph's random number to be same?

Reply:
Changed, initially I wasn't working on that op, but after deeper insight I found the seed in attrs, I think it suits the need well.